src/convert-package.ts

```diff
@@ -50,8 +50,39 @@ type ConvertPackageOptions = AnalysisConverterOptions&{
   cleanOutDir?: boolean;
 };
 
+
+/**
+ * Create and/or clean the "out" directory, setting it up for conversion.
+ */
+async function setupOutDir(outDir: string, clean: boolean) {
+  if (clean) {
+    rimraf.sync(outDir);
+  }
+  try {
+    await fs.mkdir(outDir);
+  } catch (e) {
+    if (e.errno === -17) {
+      // directory exists, do nothing
+    } else {
+      throw e;
+    }
+  }
+}
+
+/**
+ * Create and/or clean the "out" directory, setting it up for conversion.
+ */
+async function writeFile(outPath: string, newSource: string|undefined) {
+  mkdirp.sync(path.dirname(outPath));
```

Comment:
you could promisify this

Reply:
Yup, just realized convert-workspace is doing that